### ISSUE TITLE
如何复现Figure1的t-SNE图

### ISSUE THREAD

**Issue by Air000** [Syliz517/CLIP-ReID#63]
我试图复现Figure1的t-SNE图，发现text feature和image feature不聚拢。我用的是CLIP-ReID model在VeRi776训练60epoch的模型在测试集取特征，text_features = model(label=selected_ids_tensor, get_text=True)和image_features = model(imgs, get_image=True)，出来的结果如图1。
我的问题是：1. Figure1用的是训练集还是测试集？是不是用测试集的原因才聚拢得不理想？；2. 我这样取的特征对吗？还是应该拿那个拼接后的图像特征（img_feat = model(imgs, cam_label=None, view_label=None))，我试了这种，由于text_features 和image_features维度不一致，我用PCA压缩了，结果如图2？

图1（image_feature is image_features_proj[:,0]）: ![Image](https://github.com/user-attachments/assets/37208162-afa8-4a2e-a59b-9f42b62d6204)

图2（image_feature is torch.cat([img_feature, img_feature_proj], dim=1)）![Image](https://github.com/user-attachments/assets/96fe5a39-f4af-4fad-b5ca-6bbafdaff00b)
